project/src/js/socket.js

```diff
@@ -52,6 +52,23 @@ function createLobbySocket(lobbyVersion, onOpen, handlers) {
   return socketInstance;
 }
 
+function createFriendsSocket(friendsVersion, onOpen, handlers) {
```

Comment:
Since it can be a multipurpose socket, I suggest a more generic name: `createSocket(name, onOpen, handlers)`

`version` here is not an api version, rather a event version. For now I guess you can just omit the parameter and pass `0` to StrongSocket.